pulp_ansible/app/galaxy/v3/filters.py

```diff
@@ -0,0 +1,112 @@
+from django.contrib.postgres.search import SearchQuery
+from django.db.models import fields as db_fields
+from django.db.models.expressions import F, Func
+from django_filters import filters
+from django_filters import FilterSet
+from django.db.models import Q
+
+
+class CollectionVersionSearchFilter(FilterSet):
```

Comment:
need to be able to filter by repository labels too. You can use this filterset: https://github.com/pulp/pulpcore/blob/main/pulpcore/app/viewsets/custom_filters.py#L325. Might need to add pulp labels in the postgres view